components/locid_transform/src/provider/mod.rs

```diff
@@ -87,3 +89,24 @@ pub struct StrStrPair<'a>(
     #[cfg_attr(feature = "serde", serde(borrow))] pub Cow<'a, str>,
     #[cfg_attr(feature = "serde", serde(borrow))] pub Cow<'a, str>,
 );
+
+#[zerovec::make_varule(StrStrStrPairVarULE)]
+#[zerovec::derive(Debug)]
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Deserialize),
+    zerovec::derive(Deserialize)
+)]
+#[cfg_attr(
+    feature = "datagen",
+    derive(serde::Serialize, databake::Bake),
+    zerovec::derive(Serialize),
+    databake(path = icu_locid_transform::provider),
+)]
+/// A triplet of strings with a EncodeAsVarULE implementation.
+pub struct StrStrStrPair<'a>(
```

Comment:
Is this type required? `Language` is `Sized` and therefore `ULE`, so `(Language, StrStrPair)` should work?

Reply:
I got:
```
179 | #[derive(Debug)]
    |          ----- in this derive macro expansion
...
184 |     pub language_variants: VarZeroVec<'data, UnvalidatedLanguageVariantsPai...
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `VarULE` is not implemented for `(icu_locid::subtags::Language, StrStrPairVarULE)`
```

Reply:
I'm fine adding this impl or @robertbastian's suggested impl, but I wanted to observe that this is a misnomer. It is a `LangaugeStrStr`, not a `StrStrStr`.

Reply:
Agreed on the misnomer

Reply:
This misnomer is still not fixed